articles/anomaly-detection/passwords-checking.md

```diff
@@ -0,0 +1,229 @@
+Bulk Passwords Checking
+=======================
+
+In Auth0 we focus on bring you the tools you need to keep your systems and your users safe.
+Bulk passwords checking is a tool that allows you to check your user's passwords against pre-defined list of
+passwords to test their strength.
+
```

Comment:
The bulk password checking tool allows you to check your users' passwords against a pre-defined list of passwords to test their strength.

(The *users'* is intentional -- give possession to the plural user)